### ISSUE TITLE
Task list done 

### ISSUE THREAD

**Issue by vasimak** [streetwriters/notesnook#4159]
### What problem are you facing?

I use task list as to do things like market and stuff and i use the done undone button a lot. So when i want to undone something i have to search through my list, and when  i undone this task it stays in that position it was, and it isn't going up, only after i click the button move all checked tasks at the bottom.

### What's the solution you'd like to see in Notesnook?

I would like to see two separate lists, with checked and unchecked boxes like the task list in google keeps. And an auto suggestions prompt with the tasks i have already wrote when i am writing a new task , so it is not necessary to search through the list. You could see for further information how the google keeps tasklist work.
